config/vufind/searches.ini

```diff
@@ -898,9 +898,12 @@ content[] = FacetList
 recordAccessPermission = access.api.Record
 searchAccessPermission = access.api.Search
 
-; This is the maximum number of results that can be returned in a single response:
+; This is the maximum number of results that can be returned in a single response.
+; Applies to searches not using resumption token:
 maxLimit = 100
-
+; Uncomment to change the maximum amount of records returned when using resumption token in REST API.
+; Default is 200.
+;cursorMaxLimit = 500
```

Comment:
As far as I can see this is always used, so the name should rather be `cursorLimit`.